requirements.txt

```
@@ -1,6 +1,6 @@
 requests>=2.27.1
-jsonpickle~=1.4, >= 1.4.1
-cachecontrol~=0.12.6
+jsonpickle>= 1.4.1

```

```suggestion
jsonpickle>=1.4.1
```
Just to make it all match